packages/graphql/src/graphql/input-objects/generic-operators/BooleanScalarFilters.ts

```diff
@@ -28,3 +28,12 @@ export const BooleanScalarFilters = new GraphQLInputObjectType({
         },
     },
 });
+
+export const BooleanListFilters = new GraphQLInputObjectType({
```

Comment:
Do you know if this is something we currently support? if not we can keep it out of the scope for now!

Reply:
Good point

Reply:
Actually, we do support these on Cypher fields, but only equals, so I'll update this